sql/src/main/java/io/crate/operation/reference/sys/node/SysNodesExpressionFactories.java

```diff
@@ -40,8 +40,17 @@
     public SysNodesExpressionFactories() {
     }
 
+    private static Map<ColumnIdent, RowCollectExpressionFactory> sysNodeTableInfoFactories;
+
     public static Map<ColumnIdent, RowCollectExpressionFactory> getSysNodesTableInfoFactories() {
-        return ImmutableMap.<ColumnIdent, RowCollectExpressionFactory>builder()
+        if (sysNodeTableInfoFactories == null) {
+            buildSysNodesTableInfoFactories();
```

Comment:
not sure if there is a point in making this lazy.. I think it is called always on a node startup